### PR TITLE
[doc] use Brewfile to install dependencies by brew

### PR DESCRIPTION
**Pull request by mikz**
`Brewfile` is a way to define a project's dependencies and then just install them by one command `brew bundle`.

### CODE REVIEW

File: ostia-operator/Brewfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+brew 'operator-sdk'
```
Comment (jmprusi) [inline ostia-operator/Brewfile]: Can you pin a version?

Comment (mikz) [reply ostia-operator/Brewfile]: I don't think so. But you can't do that in brew anyway. There is always just the latest.

Comment (jmprusi) [reply ostia-operator/Brewfile]: Ok.. let's see how it goes :)

**Comment by unleashed** [comment]
@mikz why not use docker?

**Comment by mikz** [comment]
@unleashed because native development works pretty well.

**Pull request merged**